features/step_definitions/networking.rb

```diff
@@ -696,3 +685,35 @@
     raise "Fail to delete the ovs pod"
   end
 end
+
+Given /^the default interface on nodes is stored in the#{OPT_SYM} clipboard$/ do |cb_name|
+  ensure_admin_tagged
+  step "I select a random node's host"
+  node_name ||= node.name
+  cb_name = "interface" unless cb_name
+
+  sdn_pod = BushSlicer::Pod.get_labeled("app=sdn", project: project("openshift-sdn", switch: false), user: admin) { |pod, hash|
+    pod.node_name == node_name
+  }.first
+  cache_resources sdn_pod
+  @result = sdn_pod.exec("bash", "-c", "ip route show default | awk 'NR==1{print $5}'", as: admin)
+  raise "Failed to retrieve interface name from sdn pod" unless @result[:success]
+  cb[cb_name] = @result[:response].strip
+  logger.info "The node's default interface is stored in the #{cb_name} clipboard."
+end
+
+Given /^CNI vlan info is obtained on the#{OPT_QUOTED} node$/ do | node_name |
+  ensure_admin_tagged
+  node = node(node_name)
+  host = node.host
+  @result = host.exec_admin("/sbin/bridge vlan show")
+  raise "Failed to execute bridge vlan show command" unless @result[:success]
+end
+
+Given /^the bridge interface named "([^"]*)" is deleted from the "([^"]*)" node$/ do |bridge_name, node_name|
+  ensure_admin_tagged
+  node = node(node_name)
+  host = node.host
+  @result = host.exec_admin("/sbin/ip link delete #{bridge_name}")
```

Comment:
whatever creates these bridges and vlans, isn't it running inside a pod? If so, can we deleted bridges and VLANs from pods instead of node hosts?